docs/Charting-Concepts/SankeyChart.md

```diff
@@ -0,0 +1,112 @@
+### The chart is in a loading state because of one or more reasons below.
+- The chart data is too heavy.
+- Chart is waiting for data from a webservice.
+### Extremely high density:
+The number of nodes  in the chart are more considering the height and width of the canvas provided.
+
+## Some notable PRs and their brief description
+[PR](https://github.com/microsoft/fluentui/pull/25455)
+
+## Design figma
+Sankey Chart Figma : [Data viz (Archive) – Figma](https://www.figma.com/file/WOoCs0CmNYZhYl9xXeCGpi/Data-viz-(Archive)?type=design&node-id=21212-72120&mode=design&t=uSjYIeUd9e3vBPuc-0)
+
+## Rendering details
+The Sankey chart in SankeyChart.base.tsx is rendered using D3.js, a popular JavaScript library for data visualization. The chart is created by rendering a series of SVG elements, including rectangles for the nodes and paths for the links between the nodes.
+The chart is created in the _createSankey function, which initializes the D3 Sankey layout and sets the chart's dimensions and margins. The function then creates the nodes and links for the chart using the d3.sankey layout and the chart's data.
+The nodes are rendered as rectangles using the rect SVG element, with the position and size of each rectangle determined by the x, y, width, and height properties of the node. The links between the nodes are rendered as paths using the path SVG element, with the position and shape of each path determined by the x0, y0, x1, y1, and width properties of the link.
+The chart also includes event handlers for mouseover and mouseout events on the nodes and links, which highlight the corresponding elements and display tooltips with additional information.
+
+## Interactions
+There are various interactions that a user experiences in Line Chart which are
+#### Hover Over Nodes
+When the user hovers over nodes, the entire flow associated with that node will be highlighted, encompassing both the links and nodes within the entire flow. Furthermore, the entire flow will be colored to match the node upon which the user has hovered. To enhance visibility, all other nodes will be rendered in white with reduced opacity.Below is an example. Here user has hovered over node with value 62 . 
+![image.png](images/SankeyChartPic5.png)
+
+Hovering on the node with height less than 24px will display a callout with details of node name and weight. 
+#### Hover Over links
+When the user hovers over a link, the entire flow will be highlighted. All nodes will retain their original colors, while the links themselves will adopt a gradient color derived from the source and target nodes. Additionally, a callout will appear, displaying details about the link, such as the source node's name, weight, and the target node's name. To enhance the visual appeal, the link border will also transition into a gradient when the link is hovered over. Below is an example of a link being hovered over, in this case, between node 4 and node 12.
+
+![image.png](images/SankeyChartPic6.png)
```

Comment:
put the images in docs/assets/images folder

Reply:
Done